microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension/install.md

```diff
@@ -0,0 +1,285 @@
+## Step 2: Install Microsoft Edge
+
+Microsoft Edge is required, for the DevTools extension for VS Code.
+
+On Windows, Microsoft Edge is installed.  On macOS or Linux, install Microsoft Edge as follows:
+
+1. Go to the [Edge](https://www.microsoft.com/edge) page at Microsoft.com.
+
+
+<!-- ====================================================================== -->
+## Step 3: Install the Microsoft Edge DevTools extension
+
+Install the Microsoft Edge DevTools extension for Visual Studio Code, as follows:
+
+1. Open Visual Studio Code.
+
+1. In the Activity Bar on the left, click the **Extensions** (![Extensions icon](./get-started-images/extensions-icon.png)) button.  Or, press `Ctrl`+`Shift`+`X` on Windows/Linux or `Command`+`Shift`+`X` on macOS.  The **Extensions** Marketplace pane opens.
+
+1. In the **Search Extensions in Marketplace** text box, enter **Microsoft Edge Tools for VS Code**.
+
+1. Select the **Microsoft Edge Tools for VS Code**, and then click the **Install** button:
+
+   ![Installing the Microsoft Edge DevTools extension for Visual Studio Code](./get-started-images/vscode-edge-tools-install.png)
+
+As an alternative approach, you can use the browser to download the Microsoft Edge DevTools extension from the Visual Studio Marketplace website.  Go to [Microsoft Edge Developer Tools for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=ms-edgedevtools.vscode-edge-devtools).
+
+
+<!-- ====================================================================== -->
+## Step 4: Install Node.js and Node Package Manager (npm)
+
+To get live (realtime) analysis of your code to indicate issues, such as with wavy underlines, and to offer Quick Fixes, you must install Node.js and Node Package Manager (npm).
+
+The DevTools extension shows a popup suggestion to install Node.js and npm.  The suggestion text is similar to: "Install Node.js & npm? (suggested because you installed this extension)".
+
+1. Click the link in the popup, if it opens.
+
+1. Install Node.js and npm from [Node.js](https://nodejs.org) > Downloads > LTS (long-term stable) (or, Current).
+
+   On Windows, the .msi file is downloaded, such as `node-v16.17.1-x64.msi`.
+
+1. In the browser's Downloads pane, Click **Show in folder**.  Run the downloaded file.  The Node.js Setup Wizard opens.  Click the **Next** button.  Follow the prompts.  
+
+   Confirm that Node.js and npm are installed:
+
+1. In Visual Studio Code, select **View** > **Terminal**.  At the command prompt, enter `npm version`.  A version number is displayed for `npm` and for `node`, indicating that Node Package Manager and Node.js are installed, to support inline and live issue analysis:
+
+   ```
+   $ npm version
+   {
+   npm: '8.15.0',
+   node: '16.17.1',
+   ...
+   }
+   ```
+
+See also:
+* [Inline and live issue analysis](./inline-live-issue-analysis.md)
+
+
+<!-- ====================================================================== -->
+## Step 5: Clone the Demos repo
```

Comment:
It feels overkill to have to clone the whole demos repo when all we need is the to-do demo. Shall we create a zip of it and make that downloadable instead? This could be easily scripted to generate automatically. @captainbrosset

Reply:
It could be good to add instructions on how to download only a small zip only of the demo-to-do directory.  In addition to keeping the cloning instructions, which maintainers of this doc-set need.  In this PR or a next PR.

Reply:
I'm pushing a commit with a draft using that link.